src/Cake/Scripting/Roslyn/Stable/DefaultRoslynScriptSessionFactory.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Cake.Core;
+using Cake.Core.Configuration;
+using Cake.Core.Diagnostics;
+using Cake.Core.Scripting;
+
+namespace Cake.Scripting.Roslyn.Stable
+{
+    using Core.IO;
```

Comment:
Any special reason this isn't by the other using statements above? If not move up so all in one place.